### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by sealabr** [Shahrukh5995/EmbeddedSignup#1]
Hello,

I saw your repo in github, you were able to connect a whatsapp business account?

i'm getting:

 You don't have any WhatsApp Business Accounts.

I posted my question in stackoverflow:

https://stackoverflow.com/questions/74030145/facebook-embed-login-you-dont-have-any-whatsapp-business-accounts/74116538#74116538

can you help?

thanks.

**Comment by Shahrukh5995** [comment]
Yes this is working for me perfectly. Try deleting your whatsapp business
account and create again using the embedded signup.

**Comment by sealabr** [comment]
okay, thanks!

Em qui., 20 de out. de 2022 às 15:57, Shahrukh5995 ***@***.***>
escreveu:
